packages/docsify-changelog-plugin/src/index.js

```diff
@@ -33,11 +33,213 @@ const plugin = (hook, vm) => {
 
   hook.ready(function () {
     // Called when the script starts running, only trigger once, no arguments,
-    if (!!vm.config.changelog && !vm.config.loadNavbar) {
+    if (!!vm.config.changelog.path && !vm.config.loadNavbar) {
       loadDoc()
     }
   })
-
+  hook.mounted(function() {
+    // Place the setting inject into the CSS here
+    // We need to first build the entire settings object
+    var u = {
```

Comment:
dont use single char for variable names. 
name them something which makes it give some details. It will help in future. 

we have terser for minification so we dont have to do these things

Reply:
ok renamed to SetParams and UserParams